back/src/routes/airtable.ts

```diff
@@ -59,29 +59,41 @@ airtableRouter.post('/application_status', (req, res, next) => {
     });
 });
 
-airtableRouter.get("/isLoggedIn", function (req, res, next) {
+airtableRouter.get("/isLoggedIn", function (req, res, next) { 
   if (req.user) {
     res.send(req.user);
   } else {
     res.send("False");
   }
 });
 
-airtableRouter.post('/signout', function(req, res, next) {
-  req.logout();
-  res.send("Success");
+airtableRouter.post('/signout', function (req, res, next) {
+  req.logout()
+  res.end()
 });
 
 
+
+
 airtableRouter.post("/getInfo", function (req, res) {
   if(req.body.userName == undefined){
     res.end();
     return;
   }
-  try {
-    
-    const userName = req.body.userName;
+
+  if (!req.user) {
+    res.end();
+    return
+  }
   
+  // combs user field for username information to compare to name on the request
+  const userName = JSON.stringify(req.user).match(/(?<="Username":")([A-Za-z0-9])+/)[0]
```

Comment:
Is there a reason we can't do `req.user[0].fields.Username`?

Reply:
I didn't know that was how I should access it thank you! That cumbersome regex is a work around. I'll incorporate that and update the PR. Thanks Mack!

Reply:
Sure thing!